scrapi/util.py

```diff
@@ -51,3 +55,32 @@ def json_without_bytes(jobj):
         if isinstance(v, six.binary_type):
             jobj[k] = v.decode('utf8')
     return jobj
+
+
+def doc_to_normed_dict(doc):
```

Comment:
should this be part of the migrations?  I don't see it being useful outside of that scenario. Unless I missed something?